apps/dashboard/src/core-ui/batch-upload/upload-step.tsx

```diff
@@ -33,22 +33,10 @@ export const UploadStep: React.FC<UploadStepProps> = ({
           <div className="aspect-square w-full md:w-1/2">
             <div
               className={cn(
-                "rounded-md cursor-pointer border border-border h-full hover:border-blue-500",
+                "h-full cursor-pointer rounded-md border border-border hover:border-blue-500",
                 hasFailed ? "bg-red-200" : "bg-card",
               )}
               {...getRootProps()}
-<<<<<<< HEAD
-              cursor="pointer"
-              bg={hasFailed ? "red.200" : "inputBg"}
-              _hover={{
-                bg: hasFailed ? "red.200" : "inputBgHover",
-                borderColor: "blue.500",
-              }}
-              borderColor="inputBorder"
-              borderWidth="1px"
-              h="100%"
-=======
->>>>>>> 4a84e9e04 (temporarily revert Center eslint)
```

Comment:
This needs to be removed in the stack below - not in this PR

Reply:
should be resolved. thanks